src/XeroPHP/Remote/Query.php

```diff
@@ -105,16 +161,46 @@ public function offset($offset = 0) {
     }
 
     /**
+     * @param bool $includeArchived
+     * @return $this
+     */
+    public function includeArchived($includeArchived = true) {
```

Comment:
That makes sense.  I guess it was just tripping me up why it would sometimes be null.